opm/simulators/wells/StandardWell_impl.hpp

```diff
@@ -1183,7 +1185,7 @@ namespace Opm
         constexpr double stopped_factor = 1.e-4;
         // use stricter tolerance for dynamic thp to ameliorate network convergence
         constexpr double dynamic_thp_factor = 1.e-1;
-        if (this->stopppedOrZeroRateTarget(summary_state, well_state)) {
+        if (this->stoppedOrZeroRateTargetIndividual(summary_state, well_state)) {
             tol_wells = tol_wells*stopped_factor;
```

Comment:
I think it's a bit strange that zero target wells should be solved with a much stricter tollerance than others. For convenience, I've also skipped checking for zero targets from groups here.

Reply:
There might be different approaches to address this, but it improves the results for some problematic cases. 

https://github.com/OPM/opm-simulators/pull/4572
https://github.com/OPM/opm-simulators/pull/4556

Possibly we should use stricter tolerance by all.

Reply:
Thanks, would be happy to discuss stricter well-tolerances overall.  Especially for standard wells I think this is a good idea!